src/components/github-htmlview.component.js

```diff
@@ -48,10 +48,7 @@ const styles = {
   code: codeStyle,
   span: textStyle,
   p: { ...textStyle, margin: 0, padding: 0 },
-  sub: {
-    ...hrStyle,
-    fontSize: normalize(9),
-  },
+  sub: textStyle,
```

Comment:
sorry buddy, what I meant was:

```js
sub: {
  ...textStyle,
  fontSize: normalize(9),
}
```

We still want the text to be smaller, and we inherit from textStyle, instead of hrStyle (which inclues a border-bottom that we don't want here)

Reply:
ohh sorry, miss understanding.